hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -413,7 +413,7 @@ def consider_new_buffer(self, buffer):
         buffer = bytes(buffer)
         return buffer.startswith(self.buffer) or self.incorporate_new_buffer(buffer)
 
-    def incorporate_new_buffer(self, buffer):
+    def incorporate_new_buffer(self, buffer):  # pragma: no cover
```

Comment:
I don't particularly feel like writing covering tests for a function that is imminently slated for removal, but will do so on request.